src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -59,10 +59,33 @@ public static void main(String... args) throws IOException {
         optionParser.accepts("outputDirectory").withRequiredArg();
 
     OptionSet options = optionParser.parse(args);
+    String output = options.valueOf(outputDirectoryOption);
+    performMinimization(
+        options.valueOf(rootOption),
+        options.valuesOf(targetFilesOption),
+        options.valuesOf(jarPath),
+        options.valuesOf(targetMethodsOption),
+        options.valueOf(outputDirectoryOption));
+  }
 
-    String root = options.valueOf(rootOption);
-    List<String> targetFiles = options.valuesOf(targetFilesOption);
-    List<String> jarPaths = options.valuesOf(jarPath);
+  /**
+   * This method provides a simplified way for users to perform the minimization job using Specimin.
+   * Users can call this method directly instead of invoking the main method of Specimin to initiate
+   * the minimization process.
```

Comment:
This comment needs to mention that this is intended as an API for users who want to use Specimin as part of their own project. You've got most of that in the comment, but you should come out and say it directly. In particular, the phrases "API" and "use Specimin as a library" both need to appear, because those are what potential users will search for.